packages/zqlite/src/db.test.ts

```diff
@@ -102,7 +102,7 @@ test('slow queries are logged', () => {
         method: 'iterate',
         type: 'total',
       },
-      ['Slow query', 200],
+      ['Slow SQLite query', 200],
```

Comment:
Not quite.

`method: iterate` includes the cost of the downstream IVM pipeline rooted at the given SQLite query.

So

```ts
issue.related('labels').related('comments').related('owner');
```

could log:

```
SLOW QUERY `SELECT * FROM issue` METHOD `iterate`
```

and that includes the total cost of the entire pipeline.

```
SLOW QUERY `SELECT * FROM comment WHERE issue_id = ?`
```

would be the total time to iterate the rows coming out of the `comment` branch of that query.

Reply:
It confuses everyone so we can also remove the `iterate` logging.